tests/test_ms_ferrybox_nivatest.py

```diff
@@ -0,0 +1,30 @@
+import datetime as dt
+import json
+
+import pytest
+
+import pyniva
+
+TSB_URL = 'https://api-test.niva.no/v1/tsb/'
+META_URL = 'https://api-test.niva.no/v1/metaflow/'
+
+
+@pytest.mark.token_and_ms_ferrybox
+def test_ms_ferrybox_for_newly_populated_data():
+    """ Testing that the ms ferrybox api responds correctly to requests for newly inserted data"""
+    header = pyniva.token2header("/home/roald/PycharmProjects/other_nivacloud_stuff/"
+                                 "secrets/nivatestdev-1b5ad93f6938.json")
```

Comment:
TODO: pass in token as env variable or path to file containing token. I think just passing in the token as env variable is easiest when setting up circleci secrets?